app/src/Security/LoginFormAuthenticator.php

```diff
@@ -123,11 +123,43 @@ private function getLoginImap(User $user, string $password): bool
                     }
                 } catch (ConnectionFailedException $exc) {
                     $this->logger->error(
-                        "User cannot connect \t (Error " . $exc->getCode() . ")\t" . $exc->getMessage()
+                        sprintf(
+                            "IMAP connection failed for user '%s' on host '%s:%s' (Error %d, %s): %s",
+                            $login,
+                            $connector->getImapHost(),
+                            $connector->getImapPort(),
+                            $exc->getCode(),
+                            get_class($exc),
+                            $exc->getMessage()
+                        ),
+                        [
+                            'exception' => $exc,
```

Comment:
Il n’y a pas besoin d’un `sprintf` ici parce que le logger permet déjà de faire du remplacement de placeholder:

```suggestion
                        'IMAP connection failed for user "{imap_login}" on host "{imap_host}:{imap_port}": {message}',
                        [
                            'exception' => $exc,
                            'message' => $exc->getMessage(),
```